srtcore/core.cpp

```diff
@@ -8291,7 +8313,7 @@ int CUDT::processData(CUnit *in_unit)
         if (!m_pRcvLossList->find(packet.m_iSeqNo, packet.m_iSeqNo))
             rexmit_reason += "REQUEST";
         else
-            rexmit_reason += "ACK-TMOUT";
+            rexmit_reason += "NAKREPORT";
```

Comment:
NAKREPORT is a retransmission request.
`CRcvLossList::find(..)` returns true if the sequence is in the loss list.
If it is in the list, we have reported this loss, so probably `rexmit_reason += "NAKREPORT"`.
But is it is not in the list, why `rexmit_reason += "REQUEST"`? It should be `ACK-TMOUT`, I guess.

Reply:
Right. Moreover, when the loss wasn't found in the loss record, it can mean two things:
 - sender retransmitted forcefully (FASTREXMIT/LATEREXMIT)
 - sender retransmitted more than once and this one follows a successful retransmission

These situations can be distinguished by checking if adding the packet succeeded, but since introduction of the packet filter it became complicated and not worth specific code just for this information for the logs. I think "UNREQUESTED" or possibly "BLIND" would be then better for the "not found in the loss list" case.